### PR TITLE
EUI-7449 Grouped hearings of different linked cases for the same slot fails

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7449

### Change description ###
Grouped Hearings of different linked cases for the same slot fails

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by GaryScottHMCTS — APPROVED** [review]

File: src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export class LinkedHearingsCheckYourAnswersComponent implements OnInit, OnDestro
   }
 
   public ngOnInit(): void {
-    this.sub = this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState), take(4)).subscribe(
+    this.sub = this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState)).subscribe(
       state => {
```
Comment (phillip-whitaker-hmcts) [inline src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts:67]: Whilst you are modifying this subscription I believe it should be upgraded to the standard subscription style: https://angular.io/guide/observables i.e. with next, error, complete. This style is now deprecated.

Comment (johnbenjamin-hmcts) [reply src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts:67]: This code change is in Angular 7, and when I try next and error it says _cannot find name 'next'_. Might be one to update after merging into Angular 11?

Comment (phillip-whitaker-hmcts) [reply src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts:67]: Hmmm really? ok... should be able to do: 

`subscribe({
  next: (x: number) => console.log('Observer got a next value: ' + x)
})`

Comment (johnbenjamin-hmcts) [reply src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts]: It worked now. I missed the curly braces :) Thank you Phil.

**Review by phillip-whitaker-hmcts — APPROVED** [review]

**Pull request merged**